vita3k/host/include/host/app_util.h

```diff
@@ -85,8 +86,8 @@ struct SceAppUtilSaveDataRemoveItem {
     uint8_t reserved[36];
 };
 
-struct SceAppUtilSaveDataMountPoint {
```

Comment:
for it, this struct with this name litterlay no exist, vita header set also double struct with just different name by error